docs/spectral_cube.rst

```diff
@@ -0,0 +1,80 @@
+###########################
+Working with Spectral Cubes
+###########################
+
+Spectral cubes can be read in using the
+`SpectralCube <https://spectral-cube.readthedocs.io/en/latest/>`_ package then
+collapsed using various methods to yield a Spectrum1D object.  A specific
+example of this workflow is demonstrated here.
+
+Note that the workflow described here is for spectral cubes that are rectified
+such that one of the axes is entirely spectral and all the spaxels have the same
+``spectral_axis`` values - i.e., case 2 in
+:ref:`specutils-representation-overview`. For less-rectified cubes,
+pre-processing steps (not addressed by specutils at the time of this writing)
+will be necessary to rectify the cubes into that form.
+
+
+Collapsing Cubes
+================
+Currently, the ``specutils`` library does not support loading three dimensional
+spectral cubes and manipulating their non-spectral axes, so parsing a spectral
+cube to a :class:`~specutils.Spectrum1D` object requires some programmatic
+manipulation using the ``spectral-cube`` library. Below is an example of
+loading a spectral cube object, collapsing the cube, and creating a
+:class:`~specutils.Spectrum1D` object.
+
+Loading a cube
+==============
+
+We'll use the example cube data found in the ``spectral-cube`` package, and
+load the data from the repository directly into a new ``SpectralCube``
+object::
+
+    >>> from astropy.utils.data import download_file
+    >>> from spectral_cube import SpectralCube
+    >>> sc = SpectralCube.read(download_file("https://github.com/radio-astro-tools/spectral-cube/raw/master/spectral_cube/tests/data/example_cube.fits"), format='fits')
```

Comment:
For future reference, any doctest that downloads stuff needs to be marked as remote data. Otherwise, you'll hit the server too much and cause unrelated failures.

xref #838